fetch.bs

```diff
@@ -966,6 +984,40 @@ for other values. If <cite>HTML</cite> changes here, this standard will need cor
 Unless stated otherwise, it is unset.
 
 <p>A <a for=/>request</a> has an associated
+<dfn export for=request id=concept-request-use-token-binding>use-token-binding flag</dfn>.
+Unless stated otherwise, it is unset.
+
+<p class="note no-backref"><a for=/>Request</a>'s
+<a for=request>use-token-binding flag</a>
+controls whether the
+user agent will send its <a for=/>token binding ID</a>
+for the <a for=request>origin</a> along with the
+<a for=/>request</a> when it transmits the
+<a for=/>request</a> to the server.
+
+<p>A <a for=/>request</a> has an associated
+<dfn export for=request id=concept-request-use-referred-token-binding>use-referred-token-binding flag</dfn>.
+Unless stated otherwise, it is unset.
+
+<p class="note no-backref"><a for=request>Request</a>'s
+<a for=request>use-referred-token-binding flag</a>
+controls whether the user agent will send the
+<a for=/>token binding ID</a>
+used by the user agent with the referring <a for=request>origin</a>,
+in addition to the <a for=/>token binding ID</a> used by the
+user agent with the <a for=request>origin</a>, when it transmits the
+<a for=/>request</a> to the server.
+
+<p>A <a for=/>request</a> has an associated
+<dfn export for=request id=concept-request-referred-token-binding-origin>referred-token-binding-origin</dfn>,
+which is an <a for=request>origin</a>. Unless stated otherwise, it is null.
+
+<p class="note no-backref"><a for=/>Request</a>'s
+<a for=request>referred-token-binding-origin</a>
+specifies the <a for=request>origin</a> used to compute the <a for=/>referred token Binding ID</a>,
+when its <a for=request>use-referred-token-binding flag</a> is set.
```

Comment:
Every redirect in a redirect chain is handled independently (the server sending the redirect response only controls disclosure to the server that it is redirecting to)

I updated this section as well as a note in the "Http-redirect-fetch" section.

Please let me know if I can clarify further.